mne/epochs.py

```diff
@@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         # pick channels
         if picks is None:
             picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
-                               stim=False, eog=False, ecg=False,
+                               stim=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
```

Comment:
On a second thought, maybe there should be a convenience function that set the channels explicitly. The question is that should it use ``'auto'`` or ``True`` for ``ref_meg``?